### ISSUE TITLE
question about logitop

### ISSUE THREAD

**Issue by hzhz2020** [adebayoj/sanity_checks_saliency#11]
Dear Author

I read your paper, it is valuable work! Also thanks for sharing the code notebook.
I was trying to follow your notebook on my own pretrained network (train with tf1)

I used **y_logits = tf.layers.dense(xx,xx, name='logit')** for computing the logit, while in your train_cnn_models.py you used  **y_logits = tf.matmul(h_fc1, W_fc2) + b_fc2**. So in your **cnn_mnist_cascading_randomization.ipynb** you used **y_logits_op = tf.get_collection('logits')[0]** to get the logit op, in my case, i should use the **"logit/BiasAdd"** or 
**"logit/MatMul"** as the y_logits_op? 

Thanks a lot!
